### PR TITLE
RDCC-2035: removing all authorised services except PRD

### PR DESCRIPTION
**Pull request by AdilOoze**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RDCC-2035

### Change description ###

removing all authorised services except PRD

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -84,7 +84,7 @@ idam:
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
     pbaFromUserAccountMap: false
   s2s-authorised:
-    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend}
+    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api}
```
Comment (shashank-rastogi) [inline src/main/resources/application.yaml:87]: dont u need rd_user_profile_api to run the functional tests?

**Review by SujeethKumarAri — DISMISSED** [review]

**Review by KOTLAPRA23 — APPROVED** [review]

File: src/main/resources/application.yaml (hunk 1 above)
Comment (AdilOoze) [reply src/main/resources/application.yaml]: Yes you're correct, added it back to authorised list here: https://github.com/hmcts/rd-user-profile-api/pull/581/commits/29c21fed3716eadddcc2548eab34f3e41323e0be

File: src/main/resources/application.yaml [hunk 2]
```diff
@@ -84,7 +84,7 @@ idam:
     url: ${S2S_URL:http://rpe-service-auth-provider-aat.service.core-compute-aat.internal}
     pbaFromUserAccountMap: false
   s2s-authorised:
-    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api,xui_webapp,finrem_payment_service,fpl_case_service,iac,aac_manage_case_assignment,divorce_frontend}
+    services: ${PRD_S2S_AUTHORISED_SERVICES:rd_professional_api,rd_user_profile_api}
```
Comment (shreedhar-hmcts) [inline src/main/resources/application.yaml:87]: we have moved this to flux , will it override here ?

Comment (AdilOoze) [reply src/main/resources/application.yaml:87]: Same change will need to be made in Flux, will create PR now

**Review by SujeethKumarAri — APPROVED** [review]

File: src/main/resources/application.yaml (hunk 2 above)
Comment (AdilOoze) [reply src/main/resources/application.yaml]: Here is the Cnp-Flux PR: https://github.com/hmcts/cnp-flux-config/pull/7671

**Review by vijayrajagopalan-hmcts — APPROVED** [review]

**Review by nitishsingla1234 — APPROVED** [review]

**Pull request merged**